modules/manage/pages/schema-reg/schema-reg-overview.adoc

```diff
@@ -33,7 +33,39 @@ See the xref:reference:cluster-properties.adoc#kafka_nodelete_topics[kafka_nodel
 
 endif::[]
 
-Redpanda Schema Registry uses the default port 8081. 
+Redpanda Schema Registry uses the default port 8081.
+
+== Serialization and deserialization
+
+With Schema Registry, producers and consumers can use a specific message format, called the wire format. The wire format facilitates a seamless transfer of data by ensuring that clients easily access the correct schema in the Schema Registry for a message.
+
+The wire format is a sequence of bytes consisting of the following:
+
+. The "magic byte," a single byte that always contains the value of 0.
+. A four-byte integer containing the schema ID.
+. The rest of the serialized message.
+
+image::shared:schema-registry-wire-format.png[alt="Schema Registry wire format"]
+
+ifndef::env-cloud[]
+In the serialization process, the producer hands over the message to a key/value serializer that is part of the respective language-specific SDK. The serializer first checks whether the schema ID for the given subject exists in the local schema cache. The serializer derives the subject name based on several xref:manage:schema-reg/schema-id-validation.adoc#set-subject-name-strategy-per-topic[strategies], such as the topic name. You can also explicitly set the subject name.
+endif::[]
+
+ifdef::env-cloud[]
+In the serialization process, the producer hands over the message to a key/value serializer that is part of the respective language-specific SDK. The serializer first checks whether the schema ID for the given subject exists in the local schema cache. The serializer derives the subject name based on several strategies, such as the topic name. You can also explicitly set the subject name.
+endif::[]
+
+If the schema ID isn’t in the cache, the serializer registers the schema in the Schema Registry and collects the resulting schema ID in the response.
+
+In either case, when the serializer has the schema ID, it pads the beginning of the message with the magic byte and the encoded schema ID, and returns the byte sequence to the producer to write to the topic.
+
+In the deserialization process, the consumer fetches messages from the broker and hands them over to a deserializer. The deserializer first checks the presence of the magic byte and rejects the message if it doesn't follow the wire format.
+
+The deserializer then reads the schema ID and checks whether that schema exists in its local cache. If it finds the schema, it deserializes the message according to that schema. Otherwise, the deserializer retrieves the schema from the Schema Registry using the schema ID, then the deserializer proceeds with deserialization.
+
+ifndef::env-cloud[]
+See also: xref:manage:schema-reg/schema-id-validation.adoc[]
```

Comment:
@rockwotj @mattschumpert Should we explain more about how this new wire format content is related to this existing doc http://localhost:5002/current/manage/schema-reg/schema-id-validation/?

Reply:
you are welcome too, could be a followup IMO, but yes noting that schema ID validation basically has the broker validating the wire format is used and the schema exists (although it does not validate the full payload)

Reply:
Included additional note